pkg/trace/api/debug_server.go

```diff
@@ -3,6 +3,9 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2023-present Datadog, Inc.
 
+//go:build !serverless
```

Comment:
Are we confident then that customers won't want the debug server when in a serverless environment then? Or is this kind of debug server never going to actually be useful since you'll be starting up and tearing down the agent before it can be used?

Reply:
Correct, at this point we do not direct customers to the debug endpoint. Using this endpoint is not straightforward since the agent is only running during a function execution. At all other times, the debug endpoint will not be available to handle requests.